contracts/migration/upgradeable/AlphaKlimaRedeemUpgradeable.sol

```diff
@@ -56,7 +56,9 @@ contract AlphaKlimaRedeemUpgradeable is ERC2771ContextUpgradeable  {
 
         KLIMA.transfer(_msgSender(), amount);
     }
-    function setTrustedForwarder(address newForwarder) public onlyOwner {
-        _trustedForwarder = newForwarder;
-    }
+
+    // Can't do this https://ethereum.stackexchange.com/a/79604
+    // function setTrustedForwarder(address newForwarder) public onlyOwner {
+    //     _trustedForwarder = newForwarder;
+    // }
```

Comment:
I'm not sure what the implications of this are, but I had to comment out in order to get it to compile as it's trying to change a private member of the parent contract